include/timer.h

```diff
@@ -62,4 +80,61 @@ static inline void Delay(int milliseconds)
 	std::this_thread::sleep_for(std::chrono::milliseconds(milliseconds));
 }
 
+static inline void DelayUs(int microseconds)
+{
+	std::this_thread::sleep_for(std::chrono::microseconds(microseconds));
+}
+
+// The duration to use for precise sleep
+static constexpr int precise_delay_duration_us = 100;
+// The tolerance to allow for sleep variation
+static constexpr int precise_delay_tolerance_us = precise_delay_duration_us; 
```

Comment:
Can this be moved inside `CanDelayPrecise()` as a `constexpr`?

Reply:
sure - even with that, do you feel this going too far to avoid magic numbers?